keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -519,15 +433,15 @@ The ClusterIP service type will be single stack, so for this case there will be
 
 #### Type NodePort
 
-The NodePort service type uses the nodes IP address, which can be dual stack, and port. If the service type is NodePort and the ipFamily is DualStack [NodePort](#configuration-of-ip-family-in-service-definitions) the load balancer can be configured as dual stack, as both families will get forwarded.
+The NodePort service type uses the nodes IP address, which can be dual-stack, and port. If the service type is NodePort and the ipFamily is DualStack [NodePort](#configuration-of-ip-family-in-service-definitions) the load balancer can be configured as dual-stack, as both families will get forwarded.
```

Comment:
@lachie83 @khenidak there is no DualStack value for ipFamily, right? Maybe this needs an update.

Reply:
Good catch. Will update